RELEASE-NOTES.md

```diff
@@ -32,6 +32,7 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 - Adds `const_iterator` support to `slam::BivariateMap` and `slam::SubMap`
 
 ### Changed
+- Integer types in `src/axom/core/Types.hpp` are deprecated because c++-11 supports their equivalents.
```

Comment:
We should add a `Deprecated` subsection for this release and move this line to it, see https://keepachangelog.com/en/1.0.0/

Reply:
It seems like a good idea, although the `Deprecated` subsection would be small or zero for some releases.
@rhornung67 ?

Reply:
I think we'd only need to add specific sections to a release when there's content.

Reply:
I agree.

Reply:
Done.